docs/upgrade/UPGRADE-unreleased.md

```diff
@@ -183,6 +183,12 @@ There you can find links to upgrade notes for other versions too.
         +    - -c
         +    - config_file=/var/lib/postgresql/data/postgresql.conf
         ```
+### Tools
+- add path for tests folder into `ecs-fix` phing target of `build-dev.xml` file ([#980](https://github.com/shopsys/shopsys/pull/980))
```

Comment:
I would like to see there why i should upgrade that. So some description what will that change give me.

Reply:
what about this ? https://github.com/shopsys/shopsys/pull/980/commits/cab86c3fb8a6ada02dd21dce4fc8f2771f1e7005